### ISSUE TITLE
Creating links

### ISSUE THREAD

**Issue by broskees** [JohnnyTheTank/apiNG-design-deadwood#10]
Is there any way to give each video within the player it's own individual link for sharing and linking purposes?

**Comment by JohnnyTheTank** [comment]
yes, easy. you can just edit the template: https://github.com/JohnnyTheTank/apiNG-design-deadwood/blob/master/dist/aping_design_deadwood.html
